examples/src/bin/basic.rs

```diff
@@ -4,7 +4,7 @@
 //! position, how to add a `button` to this `window` and how to connect signals with
 //! actions.
 
-use gio::prelude::*;
+use gtk::gio::prelude::*;
```

Comment:
`gtk::prelude` should simply re-export `gio::prelude::*` (and `glib::prelude::*` etc)

Reply:
That doesn't cause conflicts in case you have gtk::SettingsExt & gio::SettingsExt? :)

Reply:
It does, that's why it should be `gtk::GtkSettingsExt`. The rename feature exists for that very reason, and that's why it's e.g. `gst::GstObjectExt` :)

Reply:
Makes sense, I have updated the PR accordingly

Reply:
No I meant that `gtk::prelude` has a
```rust
pub use gio::prelude::*;
```

In it. So that you only need to `use gtk::prelude::*` and then get everything you need :)

Reply:
I have done exactly no?  ^^

Reply:
But then you wouldn't need `use gtk::{gio::prelude::*, prelude::*};` in the examples but just `use gtk::prelude::*;`

Reply:
Ah yes, that's an old commit :P